docs/concepts.rst

```diff
@@ -185,6 +185,9 @@ Bitshift Composition
 
 *Added in Airflow 1.8*
 
+We recommend you setting operator relationships with bitshift operators rather than ``set_upstream()``
+and ``set_downstream()``.
```

Comment:
Add this to let users know that we recommend set relationships using bitshift operators